src/test/java/jalau/at18/katas/game2028/team4/MainTest.java

```diff
@@ -0,0 +1,80 @@
+package jalau.at18.katas.game2028.team4;
+
+import jalau.at18.katas.game2048.team4.Main;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class MainTest {
+    public static final int WINNER_NUM = 2048;
+    public static final int NUMBER_TWO = 2;
+
+    @Test
+    public void moveNumbersUpTest() {
+        int[][] matrix = {{0, 0, 0, 0}, {0, NUMBER_TWO, NUMBER_TWO, NUMBER_TWO}, 
+			{0, NUMBER_TWO, NUMBER_TWO, NUMBER_TWO}, {NUMBER_TWO, 0, 0, 0}};
+        int[][] expectedMatrix = {{NUMBER_TWO, NUMBER_TWO, NUMBER_TWO, NUMBER_TWO}, 
```

Comment:
if moved up, there are some "2"s that should be merged and then expected should end up with some "4"s from merged tiles, is the merging part of this operation or is it something that is in a separated method?